cypress/e2e/ModelizerPage/TextView/file/delete.file.feature

```diff
@@ -0,0 +1,237 @@
+Feature: Test modelizer text view: delete file and folder
+
+  Background:
+    Given I clear cache
+    And I set context field "repository_url" with "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set context field "projectName" with "leto-modelizer-project-test"
+    And I visit the "/"
+
+    When I click on "[data-cy=\"import-project\"]"
+    And  I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-repository-input\"]" text "{{ repository_url }}"
+    And  I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And  I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And  I click on "[data-cy=\"import-project-form\"] [data-cy=\"import-project-form-submit\"]"
+    Then I expect "positive" toast to appear with text "Project has been imported 🥳!"
+    And  I expect "[data-cy=\"import-project-form\"]" is closed
+    And  I expect current url is "modelizer/{{ projectName }}/model"
+    And  I expect "[data-cy=\"project-name\"]" is "{{ projectName }}"
+    And  I visit the "/#/modelizer/{{projectName}}/text"
+    And  I expect "[data-cy=\"file-explorer\"] [data-cy=\"file-label-{{ projectName }}\"]" exists
+    When I wait 2 seconds
+
+  Scenario: Delete a file of the root folder should remove it from file explorer
+    When I click on "[data-cy=\"file-explorer\"] [data-cy=\"file-label-{{ projectName }}\"]"
+    When I hover "[data-cy=\"file-explorer\"] [data-cy=\"file-explorer-buttons-branch.txt\"]" to make it visible and click on it
+    Then I expect "[data-cy=\"file-explorer-action-menu\"]" exists
+    And  I click on "[data-cy=\"file-explorer-menu-delete-file\"]"
+    And  I expect "[data-cy=\"delete-file-dialog\"]" exists
+    And  I click on "[data-cy=\"delete-file-form\"] [data-cy=\"delete-file-submit\"]"
+    Then I expect "positive" toast to appear with text "File is deleted."
+    And  I expect "[data-cy=\"delete-file-form\"]" is closed
+    And  I expect "[data-cy=\"file-label-branch.txt.js\"]" not exists
+
+  Scenario: Delete last file of a sub-folder should not expand sub-folder and close active tab
```

Comment:
expect to have two different test.
Delete last file of a sub-folder should not expand sub-folder
Delete activeFile should close its tab

Reply:
done